database/migrations/2014_01_01_110000_add_timezones.php

```diff
@@ -76,17 +69,14 @@ public function up()
                     'standard'           => '-10.00',
                 ],
                 [
-                    'id'                 => 8,
                     'name'               => 'HADT',
                     'title'              => 'Hawaii no DST',
                     'php'                => 'Pacific/Honolulu',
                     'is_daylight_saving' => 0,
                     'dst'                => '-9.00',
                     'standard'           => '-9.00',
                 ]
```

Comment:
With these timezones, I want to guarantee that the ID number in the database for each of them is the same as the one here. Would that be a different approach to migrating them?

Reply:
Its extremely bad practice to rely on auto assigned DB ids in any way at all.... I'm guessing this may be related to concerns about migrating existing data to new schema, but its kind of forcing a problem unique to tger into the implementation.  It would be better for any mapping to occur during the data migration for any tables that reference timezone_id. The ids can remain, but keeping them (with the bulk approach or the row by row approach) doesn't actually guarantee that '8' will be HADT.  In either case, if 8 is already assigned to something else, the something else will remain.

Reply:
Ok, let's stick with this approach. I'll get away from my desire to hard code ID numbers. Thank you!